src/sideof.jl

```diff
@@ -59,86 +59,112 @@ Possible results are `IN`, `OUT` or `ON` the `ring`.
 function sideof(point::Point, ring::Ring)
   assertion(CoordRefSystems.ncoords(crs(point)) == 2, "points must have 2 coordinates")
 
-  v = vertices(ring)
   n = nvertices(ring)
+  if n <= 1000 || Threads.nthreads() == 1 # Serial version
+    _sideof_hao_serial(point, ring)
```

Comment:
let's follow the code style and avoid underscores, here we can use `_sideofserial`

Reply:
Suggestion implemented